### ISSUE TITLE
Feature Hotfix or Bugfix on old version

### ISSUE THREAD

**Issue by eksrha** [fullstack-devops/awesome-ci#27]

**Comment by eksrha** [comment]
Hint: ignore releases with name `Hotfix`!

**Comment by eksrha** [comment]
```javascript
try {
        // only grab releases as latest not hotfix
        release = (await repo.listReleases()).data.filter(rel => rel.name.includes("Release"))[0].tag_name
    } catch (e) {
}
```

**Comment by insidegreen** [comment]
A Hotfix or a Bugfix must increase the Bugfix version by default and never the minor or major version number.

**Comment by eksrha** [comment]
resolved by #85
